packages/forma-36-react-components/src/components/Tabs/README.mdx

```diff
@@ -7,10 +7,23 @@ github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-re
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-tabs--default'
 ---
 
-Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
+Tabs is a component that makes navigating between sections of related content possible, displaying one section at a time.
+
+## Table of contents
+
+- [How to use Tabs](#how-to-use-Tabs)
+- [Code examples](#code-examples)
+- [Content recommendations](#content-recommendations)
+
+## How to use Tabs
+
+Use Tabs if the content can be separated into sections that make sense as standalone pieces of information.
+Be aware that the users will not see all the content at the same time. Make sure the first section is the most relevant one.
+
+## Code examples
 
 ```jsx
-<Tabs role="navigation" withDivider>
+<Tabs role="tablist" withDivider>
```

Comment:
Super! thx for the contribution! 
I think we might want to add also an example with `< TabPanel />`? WDYT?

Reply:
@burakukula done ✅